File: src/main/java/net/botlify/anticaptchacom/response/solution/RecaptchaSolution.java
package net.botlify.anticaptchacom.response.solution;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.ToString;

@Getter
@ToString
@EqualsAndHashCode
public class RecaptchaSolution {

  /**
   * Token string required for interacting with the submitted form on the target website.
   */
  @JsonProperty("gRecaptchaResponse")
  private String gRecaptchaResponse;

}

File: src/main/kotlin/net/botlify/anticaptchacom/data/response/solution/RecaptchaSolution.kt
package net.botlify.anticaptchacom.data.response.solution

import com.fasterxml.jackson.annotation.JsonProperty

@Deprecated("Not implemented yet")
data class RecaptchaSolution(
    /**
     * Token string required for interacting with the submitted form on the target website.
     */
    @JsonProperty("gRecaptchaResponse")
    val gRecaptchaResponse: String,
)